migrate/migrator.go

```diff
@@ -174,39 +195,10 @@ func (m *Migrator) selectLastGroup(ctx context.Context) (*MigrationGroup, error)
 }
 
 func (m *Migrator) MarkCompleted(ctx context.Context) (*MigrationGroup, error) {
-	if err := m.validate(); err != nil {
-		return nil, err
-	}
-
-	if err := m.Lock(ctx); err != nil {
-		return nil, err
-	}
-	defer m.Unlock(ctx) //nolint:errcheck
-
-	migrations, lastGroupID, err := m.selectNewMigrations(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	if len(migrations) == 0 {
-		return new(MigrationGroup), nil
-	}
-
-	group := &MigrationGroup{
-		ID:         lastGroupID + 1,
-		Migrations: migrations,
-	}
-
-	for i := range migrations {
-		migration := &migrations[i]
-		migration.GroupID = group.ID
-		migration.Up = nil
-		if err := m.runUp(ctx, m.db, migration); err != nil {
-			return nil, err
-		}
-	}
-
-	return group, nil
+	log.Printf(
+		"DEPRECATED: bun: replace MarkCompleted(ctx) with " +
```

Comment:
Normally, dry-run means to print what will be done without making any changes.

That is, dry-run is useful, but probably should not do any changes to the DB?